ui/src/components/GoogleSignInWrapper.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import Dialog, { DialogContent } from 'material-ui/Dialog';
+import GoogleLogin from 'react-google-login';
```

Comment:
I imagine this uses https://developers.google.com/identity/sign-in/web/reference under the hood?

Reply:
Yes - exactly. It even returns the GoogleUser object described at that page. It just saves a bunch of overhead. That said, it doesn't save me from having to refresh the token myself, although maybe I'm missing a feature of the GoogleUser object.